### ISSUE TITLE
Seed security prices by default

### ISSUE THREAD

**Issue by Martin-Molinero** [QuantConnect/Lean#8255]
#### Expected Behavior

- Add new algorithm setting, enabled by default, for security seed. If enabled after a security is added we should seed an initial historical price to them
#### Actual Behavior

- Security currently do not have an initial price set
#### Potential Solution

- After universe selection, for all security additions changes, we can run them through a history request to see their initial price.
See related `GetLastKnownPrices`, currently supports a single asset, we want to run history requests in parallel -> universe selection of 500 assets can't seed sequentially
#### Reproducing the Problem

N/A
#### System Information

N/A
#### Checklist

- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)

- [x] I have provided detailed steps to reproduce the issue

**Comment by ArthurAsenheimer** [comment]
@Martin-Molinero 
I think this is a great idea and it would benefit most users. 
I just want to add a suggestion and I don't want to open a new ticket because it fits so well here. 
Could we extend the security seeder to also take into account the open interest for futures and options?

**Comment by Martin-Molinero** [comment]
See related https://github.com/QuantConnect/Lean/issues/4736

**Comment by jhonabreul** [comment]
`AlgorithmSettings.SeedInitialPrices` is false by default, opening this issue again until we can enable it by default (performance needs to be improved)
